mlrun/track/trackers/mlflow_tracker.py

```diff
@@ -122,20 +122,31 @@ def import_run(
                 "MLFlow's environment variables."
             )
         run = run[0]  # We are using a run id, so only one will be returned in the list.
-
+        # Create a run spec and metadata for creating the run object to hold the mlflow run:
         run_spec = RunSpec(function=function_name, handler=handler)
         run_metadata = RunMetadata(
             uid=run.info.run_uuid, name=run.info.run_name, project=project.name
         )
         run_object = RunObject(spec=run_spec, metadata=run_metadata)
+        # Create a context from the run object:
         ctx = mlrun.get_or_create_ctx(
             name=run.info.run_name,
             spec=run_object,
         )
+        # Store the run in the MLRun DB, then import the MLFlow data to it:
         ctx.store_run()
         cls._log_run(context=ctx, run=run, is_offline=True)
-        ctx.set_state(execution_state="completed", commit=False)
-        ctx.commit(completed=True)
+
+        # Create a rundb in order to update the run's state as completed (can't be done using context)
+        rundb = mlrun.get_run_db()
+        project = run_object.metadata.project
+        uid = run_object.metadata.uid
+        updates = {
+            "status.last_update": now_date().isoformat(),
```

Comment:
wasn't that done automatically? We saw the last update was correct without it didn't we?

Reply:
this is what alon and i did yesterday, needed to update manually in order for the run to be marked as completed and to have the green light in ui